Index: aws-sdk-cpp-1.9.43/cmake/external_dependencies.cmake
===================================================================
--- aws-sdk-cpp-1.9.43.orig/cmake/external_dependencies.cmake
+++ aws-sdk-cpp-1.9.43/cmake/external_dependencies.cmake
@@ -21,6 +21,8 @@ if(NOT NO_ENCRYPTION)
     elseif(PLATFORM_APPLE)
         set(ENABLE_COMMONCRYPTO_ENCRYPTION ON)
     endif()
+     # conda defaults: force use of OpenSSL for all platforms
+    set(ENABLE_OPENSSL_ENCRYPTION ON)
 else()
     set(ENABLE_INJECTED_ENCRYPTION ON)
 endif()
